rules/integrations/aws/impact_rds_instance_modified_or_deleted.toml

```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2021/06/06"
+maturity = "production"
+updated_date = "2021/06/06"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the modification or deletion of an Amazon Relational Database Service (RDS) Aurora database instance.
```

Comment:
This one could be elaborated on why this should be monitored

Reply:
Better Now.

Reply:
For some reason my change isn't updating, probably cause the PR is closed.

https://github.com/austinsonger/detection-rules/blob/AWS-RDS-Instance-Modified-or-Deleted/rules/integrations/aws/impact_rds_instance_modified_or_deleted.toml